gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/dsl/JvmApplicationDistributions.kt

```diff
@@ -32,4 +32,9 @@ abstract class JvmApplicationDistributions : AbstractDistributions() {
     fun windows(fn: Action<WindowsPlatformSettings>) {
         fn.execute(windows)
     }
+    
+    internal val fileAssociations: MutableSet<FileAssociation> = mutableSetOf()
+    fun fileAssociation(mimeType: String, extension: String, description: String) {
```

Comment:
Let's add a doc for this function to https://github.com/JetBrains/compose-multiplatform/tree/master/tutorials/Native_distributions_and_local_execution#platform-specific-options

(1-2 points below `installationPath`

Reply:
I extracted it to a new section as it is quite large because of an example.